awx/main/models/execution_environments.py

```diff
@@ -27,6 +26,7 @@ class Meta:
         verbose_name=_('image location'),
         help_text=_("The registry location where the container is stored."),
     )
+    pull = models.BooleanField(default=True)
```

Comment:
I would like to see this put to use, or at least a plan of how it might be used, because it's `ansible-runner` that actually uses the podman/docker commands (or not even that, with openshift). Could we leave it out until we know more?